src/editor/nodes/ImageNode.js

```diff
@@ -15,11 +15,12 @@ export default class ImageNode extends EditorNodeMixin(Image) {
   static async deserialize(editor, json, loadAsync, onError) {
     const node = await super.deserialize(editor, json);
 
-    const { src, projection } = json.components.find(c => c.name === "image").props;
+    const { src, projection, controls } = json.components.find(c => c.name === "image").props;
 
     loadAsync(
       (async () => {
         await node.load(src, onError);
+        node.controls = controls === undefined ? true : controls;
```

Comment:
Not sure which is a reasonable default : for images to have "controls" or not. For images that have been used so far as a way to texture a wall, it's not clear that the scene author wants menus to appear when you hover over the image in hubs.

Reply:
I'm going to change the default in the constructor to true and just coerce undefined to false here instead.